backends/tc/ebpfCodeGen.cpp

```diff
@@ -1347,9 +1347,9 @@ bool ControlBodyTranslatorPNA::checkPnaPortMem(const IR::Member *m) {
 }
 
 void ControlBodyTranslatorPNA::processFunction(const P4::ExternFunction *function) {
-    if (function->expr->toString() == "send_to_port" ||
-        function->expr->toString() == "drop_packet") {
-        if (function->expr->toString() != "drop_packet") {
+    if (function->expr->method->toString() == "send_to_port" ||
```

Comment:
This is an `IR::Method` where `toString` should be fine. Although I would just get the name from the declaration?

Reply:
Sorry, I misunderstood your original comment.

So, the original code is accessing `function->expr->toString()`, which previously simply invoked the `function->expr->method`'s `toString()` method (and btw `function->expr->method` is an `IR::Expression`, not an `IR::Method`). Now that `function->expr->toString()` has changed, I have modified this code to directly access `function->expr->method->toString()` instead of accessing it indirectly as before.

> In this case toString uses externalName(), which may have unexpected behavior since it may use annotations as part of the name.

This code previously already indirectly accessed this `toString()` method, so if this is a problem, then it was already there. My changes do not change the original behavior.